applications/asset_tracker_v2/tests/gps_module/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+cmake_minimum_required(VERSION 3.13.1)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(gps_module_test)
+
+# generate runner for the test
+test_runner_generate(src/gps_module_test.c)
+
+# create mock
+cmock_handle(${ZEPHYR_BASE}/../nrf/include/event_manager.h )
+cmock_handle(${ZEPHYR_BASE}/../nrf/include/date_time.h )
+cmock_handle(${ZEPHYR_BASE}/../nrf/applications/asset_tracker_v2/src/modules/modules_common.h )
```

Comment:
```suggestion
cmock_handle(../../src/modules/modules_common.h )
```

Reply:
Forgot to mention this one.

Reply:
Yeah I realized that after I pushed yesterday. I made another commit that fixes this. Tested the last push in the out of tree repo and it seems to build and run.